include/libkrun.h

```diff
@@ -95,16 +95,23 @@ int32_t krun_set_root_disk(uint32_t ctx_id, const char *disk_path);
 int32_t krun_set_data_disk(uint32_t ctx_id, const char *disk_path);
 
 /**
- * Adds a disk image to be used as a general partition for the microVM.
+ * Adds a disk image to be used as a general partition for the microVM. The only supported image
+ * format is "raw".
  *
  * This API is mutually exclusive with the deprecated krun_set_root_disk and
  * krun_set_data_disk methods and must not be used together.
  *
+ * SECURITY NOTE:
+ * By all possible means, one should avoid probing the disk image that will be given to libkrun.
+ * If a raw file is probed, and a QCOW2 header is written into the raw file's header by the guest,
+ * the next time libkrun launches the VM it will boot as a QCOW2 image. QCOW2 files can, and will,
+ * open other files. The guest will most likely have access to the data in those files, essentially
```

Comment:
Given that libkrun will no longer probe, it’s not automatically true that “the next time libkrun […] will boot as a QCOW2 image”. This is only true if the user were to re-probe the image, and then tell libkrun that it’s a qcow2 image now.

Also, this function will only ever open the image as raw; I think a comment about image formats, probing, and why this doesn’t probe makes absolute sense, but this comment as-is no longer really fits it.

Reply:
Updated the comments. Let me know what you think!

Reply:
I missed some capitalization mistakes, which I will fix in the next push.